index.html

```diff
@@ -10,7 +10,23 @@
     <div class="container">
       <h1 id="title">Welcome to Tic Tac Toe!</h1>
       <section id="game-container">
-        
+        <table>
```

Comment:
So the handlebars compiler is automatically putting the partial named `game.html` in the `game-container`, and will automatically follow that pattern moving forward. If you want you can remove that and use HTML like this, but it will probably make your testing harder. Also I have a board made already (but you should use yours) :)